FunctionHomework/test/ValidationTest.java

```diff
@@ -0,0 +1,23 @@
+
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class ValidationTest {
+
+    @Test
+    void valid() {
+        int a = 0;
+        int b = 0;
+        Assertions.assertFalse(!Validation.valid(a,b));
```

Comment:
assertFalse not is the same as:

Assertions.assertTrue(Validation.valid(a,b));